server/app/lib/fusor/validators/deployment_validator.rb

```diff
@@ -79,6 +80,19 @@ def validate(deployment)
           if !deployment.rhev_is_self_hosted && deployment.rhev_hypervisor_hosts.count < 1
             deployment.errors[:rhev_hypervisor_hosts] << _('RHEV deployments must have at least one Hypervisor')
           end
+
+          special = "?<>',?[]}{=-)(*&^%$#`~{}"
```

Comment:
The regex you had before was better. Here you are missing a bunch of characters like `|` and `\`. Also, there are tons of non-ascii characters this won't check for.